.changeset/one-bundle.md

```diff
@@ -0,0 +1,6 @@
+---
+"@bigtest/bundler": patch
```

Comment:
Since this is a breaking change, I'd make it a minor. This will actually automatically cause server to both bump a patch, and have it update its dependencies.

Reply:
very true.  Done